Assets/Scripts/UI/DraggableObject.cs

```diff
@@ -25,6 +25,9 @@ public class DraggableObject : MonoBehaviour, IDragHandler, IBeginDragHandler, I
     [SerializeField]
     [Tooltip("The position to return to if dragging ceases and no SnapLocation is snapped to.")]
     protected Vector2 defaultPosition;
+    [SerializeField]
+    [Tooltip("The size of the draggable block.")]
+    Vector3 size;
```

Comment:
Make DraggableObject have two Vector3s: One for non-dragging scale and one for dragging scale. It would be good for DraggableObject.cs to have one setter method for each of these.